tests/backend/test_handshake.py

```diff
@@ -54,9 +55,11 @@ async def test_handshake_incompatible_version(backend, server_factory):
         assert unpackb(result_req) == {
             "handshake": "result",
             "result": "bad_protocol",
-            "help": "No overlap between client API versions {3.0} and backend API versions {"
-            + str(API_VERSION)
-            + ", 1.3}",
+            "help": "No overlap between client API versions {"
```

Comment:
:/

```python
f"No overlap between client API versions {{{incompatible_version}}} and backend API version {{{', '.join(map(str, ServerHandshake.SUPPORTED_API_VERSIONS))}}}"
```

(and if you find `{{{x}}}` ugly... well that's why we are supposed to use back-tick everywhere 😄

Reply:
I don't think it's ugly, it's fine with me!

> well that's why we are supposed to use back-tick everywhere

I think curly braces are used here because we're talking about sets of versions.